cmd/commands/root.go

```diff
@@ -173,6 +176,7 @@ func NewCli() *cobra.Command {
 	rootCmd.Flags().BoolVarP(&flags.version, "version", "V", false, "Prints the version number of cpackget and exit")
 	rootCmd.PersistentFlags().BoolP("quiet", "q", false, "Run cpackget silently, printing only error messages")
 	rootCmd.PersistentFlags().BoolP("verbose", "v", false, "Sets verboseness level: None (Errors + Info + Warnings), -v (all + Debugging). Specify \"-q\" for no messages")
+	rootCmd.PersistentFlags().BoolP("encoded-progress", "E", false, "Reports encoded progress for files and download when used by other tools")
```

Comment:
from the description, I understand that this option should only be used by sub command `add`
If this is the case, this should be added in subcommand file
https://github.com/Open-CMSIS-Pack/cpackget/blob/main/cmd/commands/add.go#L124-L129
Adding to the rootcommand as persistentflag will make this option available in all the subcommand. Not sure if that is what the expectation is.

Reply:
At least Add and Update must get this.

Reply:
In that case, we have 2 options

1. Add this option in `Add` and `Update` command respectively
2. Mark `encoded-progress` option as hidden in rest of the subcommand (except `Add` and `Update`)

Reply:
fixed:
adding "encoded-progress", "E" only for commands init, add and update-index